trading_calendars/trading_calendar.py

```diff
@@ -129,15 +142,32 @@ def __init__(self, start=start_default, end=end_default):
         # Overwrite the special opens and closes on top of the standard ones.
         _overwrite_special_dates(_all_days, self._opens, _special_opens)
         _overwrite_special_dates(_all_days, self._closes, _special_closes)
+        _remove_breaks_for_special_dates(
+            _all_days,
+            self._break_starts,
+            _special_closes,
+        )
+        _remove_breaks_for_special_dates(
+            _all_days,
+            self._break_ends,
+            _special_closes,
+        )
 
         # In pandas 0.16.1 _opens and _closes will lose their timezone
         # information. This looks like it has been resolved in 0.17.1.
         # http://pandas.pydata.org/pandas-docs/stable/whatsnew.html#datetime-with-tz  # noqa
         self.schedule = DataFrame(
             index=_all_days,
-            columns=['market_open', 'market_close'],
+            columns=[
+                'market_open',
+                'break_start',
+                'break_end',
+                'market_close'
```

Comment:
IIRC as of 3.something dicts are guarenteed to be ordered, they were ordered before, but that was an implementation detail. 


I think we will get the same result here actually not specifying `columns`. Unless that isn't true in older pandas?

Reply:
Only reason I don't want to use an `OrderedDict` is because we still(?) want to support py27 and that was introduced in py 3.2. I'd rather not add a compatibility check here.

Reply:
I'm very happy leaving this as is, but just so you're aware, `OrderedDict` is `New in version 2.7.`

Reply:
My bad - I misread the docs here.